pkgs/applications/networking/instant-messengers/spectral/default.nix

```diff
@@ -8,35 +8,20 @@
 , qtmacextras
 }:
 
-let
-  # Following "borrowed" from yubikey-manager-qt
-  qmlPath = qmlLib: "${qmlLib}/${qtbase.qtQmlPrefix}";
-
-  inherit (stdenv) lib;
-
-  qml2ImportPath = lib.concatMapStringsSep ":" qmlPath [
-    qtbase.bin qtdeclarative.bin qtquickcontrols2.bin qtgraphicaleffects qtmultimedia
-  ];
-
-in stdenv.mkDerivation rec {
+stdenv.mkDerivation rec {
```

Comment:
The preferred way to do this is use `mkDerivation` instead of `stdenv.mkDerivation`, so that you do not need to call `wrapQtAppsHook` directly. `mkDerivation` also sets some Qt-specific build flags. Please refer to [the manual](https://hydra.nixos.org/build/96804884/download/1/nixpkgs/manual.html#sec-language-qt) for more information.

Reply:
Hm, I checked the docs, it seems really confusing to name them the same thing. Plus, I assume `mkDerivation` isn't soley for Qt apps, or is it?

I can take a look at this later and try to get it working with the more succinct syntax. Thanks for pointing this out and linking the docs!